plenary/plenary-14.yaml

```diff
@@ -0,0 +1,326 @@
+        ISO/TC 154 notes that this recommendation is in line with JTC1/SC30 resolution 1997/1
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 146
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 having noted the ISO/TC 154 US TAG  position with regard to the extended MoU,and in particular, the sentence « We are aware that the ISO/TMB also agreed with not extending the MoU to include CALS » have concluded that their interpretation is incorrect. ISO/TC 154 accordingly resolves that ISO/CS responds officially to the US TAG clearly setting out the correct TMB position
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 147
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: supports
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 supports the extension to the MoU (MoU/CC/97 N0022 Rev1) in line with resolution  94/1997 of the ISO/TMB
+      - type: requests
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: requests its members to provide comments on the draft extended MoU to the TC/154 secretariat by the 9th of April 1998
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 148
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: nominates
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 nominates Alain Thienot and Denis Hill to represent ISO/TC 154 at the Miami MoU/CC meeting from the 29th April to the 1st of May 1998
+
+
+  - dates:
+      - 1998-03-12
+      - 1998-03-13
+    identifier: 149
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Unanimous
+
+    actions:
+      - type: recognises
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 noted the TMB resolution 5/1998 regarding Fast-track of UN/EDIFACT directories but recognises that standardization of the full directory content is not practical
+      - type: considers
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: ISO/TC 154 considers that content such as Section 1 of the message boiler plates is a  potential candidate
+      - type: requests
+        date_effective:
+          start: 1998-03-12
+          end: 1998-03-13
+        message: CEFACT and the interested user communities, such as Banking, are requested to clearly identify what they consider relevant for standardization
```

Comment:
Here, I'm not sure about the setting in the `actions:` section. As it is shown above, I used the word `recognises` and `considers` as action types. Is that correct?

Source:
![capture10](https://user-images.githubusercontent.com/33627611/71759744-1ae70880-2e88-11ea-9aaf-29a0cbb039f4.JPG)